src/plugins/data/public/ui/filter_bar/filter_bar.tsx

```diff
@@ -95,7 +96,7 @@ function FilterBarUI(props: Props) {
           anchorPosition="downLeft"
           withTitle
           panelPaddingSize="none"
-          ownFocus={true}
+          ownFocus={props.popoverHasOwnFocus}
```

Comment:
This was causing flickering when opening a filter popover.

Reply:
Is this a bug we need to open on EUI? (A popover that acts as a focus trap can have a better experience for screen reader users.)

Reply:
I addressed (or at least tried to) here: https://github.com/elastic/kibana/pull/56222
in order not to bloat this PR and removed setting of this prop.